src/appleseed.shaders/CMakeLists.txt

```diff
@@ -156,6 +156,7 @@ source_group ("include\\appleseed\\transform" FILES
 
 set (src_blenderseed_sources
      src/blenderseed/as_closure2surface.osl
+     src/blenderseed/as_blender_areaLight.osl
```

Comment:
One last thing: can you sort this alphabetically? You can merge afterward. Thanks!

Reply:
Will do.  I don't have merge rights on this repo :)

Reply:
What?? Wait :)